### PR TITLE
Demand Progress day of action page

### PR DESCRIPTION
**Pull request by tibetsprague**
/july12?org=dp loads a page with the Demand Progress disclaimer, contact info and a form that submits directly to ActionKit

### CODE REVIEW

**Comment by mikemorris** [comment]
@tibetsprague Biggest flag here is that `js/battle.js` is a generated file - changes should be applied to the source files in `_src/js/`, and concatenate/minified by running `npm install && npm start` in `_src`.

Sorry if this is confusing because the build system was rewritten for the homepage redesign (so we currently have two build systems and should start migrating things over to the new one as any pages get redesigned, but leaving things in place for the current `/july12` page is okay for now).

Aside from that looks alright, would be good to start thinking about how to possibly integrate this into the new structure in https://github.com/fightforthefuture/battleforthenet/blob/gh-pages/ts/petition-form.tsx for an upcoming update to the day of action page.

**Comment by tibetsprague** [comment]
oops, ok @mikemorris fixed that

File: templates/SignupFormDP.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      </section>
+    </main>
+
+    <!-- This field gets updated after geocoding IP address -->
+    <input type="hidden" name="country" value="US"/>
+
+    <footer>
+      <button type="submit">Join the Protest</button>
+      <div class="disclaimer">
+        We’ll send you everything you need to participate. <a href="https://demandprogress.org/" target="_blank">Demand Progress</a></span> will email you about work related to this, and you can unsubscribe at any time. If you enter your number (it's optional) Fight for the Future may follow up by SMS, at which point message &amp; data rates apply. You can always text STOP to stop receiving messages. Here’s our <a href="https://demandprogress.org/privacy-policy/" target="_blank">Privacy Policy</a>.
```
Comment (mikemorris) [inline templates/SignupFormDP.html]: Just confirming that we still want to collect phone numbers on the DP version, and keeping the SMS disclaimer? If so, should change the remaining mention of `Fight for the Future`

File: july12/index.html [hunk 2]
```diff
@@ -297,6 +298,12 @@
 
     window.optimizely || (window.optimizely = []);
 
+    var params = new URLSearchParams(window.location.search.substring(1));
+    if (params.get('org') == 'dp') {
+      global.demandProgress = true;
```
Comment (mikemorris) [inline july12/index.html]: Is this being used anywhere? Would prefer to avoid adding anything more to global namesapce unless absolutely necessary.

Comment (tibetsprague) [inline july12/index.html]: no its not right now, will remove

File: templates/SignupFormDP.html (hunk 1 above)
Comment (tibetsprague) [inline templates/SignupFormDP.html]: hmm, let me double check

**Comment by mikemorris** [comment]
Was switching away from the thanks modal to the text replacement intentional?

screen shot 2017-06-26 at 4 29 20 pm

**Comment by tibetsprague** [comment]
No, it came from switching to the actionkit JSONP form submission stuff. i can look into popping up the modal when that is done, but probably good to get this out first. Fixed the other two things

**Comment by mikemorris** [comment]
Gotcha, I wouldn't worry about popping up the modal in that case, as we'll probably want to move all this over to the new architecture at some point, and could address that then.

**Pull request merged**